projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1443,3 +1452,12 @@ protected static function build_sprintf( $args ) {
 function publicize_calypso_url() {
 	return Redirect::get_url( 'calypso-marketing-connections', array( 'site' => ( new Status() )->get_site_suffix() ) );
 }
+
+/**
+ * Get Jetpack Cloud URL for Publicize connections.
+ *
+ * @return string
+ */
+function jetpack_social_connections_classic_editor_url() {
```

Comment:
This is probably a question of style, but should we combine these, and have a `get_connections_url` function that could take a parameter, that would adjust the source accordingly? We might want to have `publicize_calypso_url` then call it.